README.md

```diff
@@ -1,88 +1,135 @@
+NPM package. Any deviations from the patterns defined there should be
+implemented in the client application.
 
-Until this project is published to NPM, you have two options to get it included as a dependency of your Node compatible application.
+Usage
+----------
 
----
+There are two options for including crds-styles in your project.
 
-### Option #1 - Symlink Local Directory
+### Option #1 (Recommended) - External Build
 
-Assuming the `crds-styles` repo resides in the same local directory as your project, you can do the following to setup the symlink...
+The easiest way to include crds-styles is by loading the external CSS file. The
+build is deployed to S3 and is available through our Cloudfront distribution.
 
-    $ cd path/to/your/project-directory
-    $ npm link ../crds-styles
+This approached is most useful when you don't need to override any styles, or
+when the styles you're overriding don't require needing crds-styles' (or
+Bootstrap's) sass variables.
 
-Then update your project's `package.json` file with a reference to `crds-styles`, for example...
+```html
+<link rel="stylesheet" href="//d1tmclqz61gqwd.cloudfront.net/styles/crds-styles-2.0.0.min.css">
+```
 
-    {
-      "name": "foo",
-      "version": "0.0.0",
-      "dependencies": {
-        "crds-styles": "*"
-      }
-    }
+_(Note: Replace `2.0.0` with the desired version.)_
 
----
+### Option #2 - NPM via package.json
 
-### Option #2 - Reference Github URL in package.json
+If your project needs to use crds-styles' sass variables or requires that you
+build crds-styles locally, you can bring it in via NPM in your `package.json`
+file.
 
-If you don't have the DDK cloned locally, you can just reference the repo directly in your `package.json` file, like so...
+```json
+{
+  "name": "my-project",
+  "version": "1.0.0",
+  "dependencies": {
+    "crds-styles": "2.0.0"
+  }
+}
+```
 
-    {
-      "name": "foo",
-      "version": "0.0.0",
-      "dependencies": {
-        "crds-styles": "crdschurch/crds-styles"
-      }
-    }
+You can also use a reference to the GitHub URL in cases where you would like to
+use upstream changes that haven't yet been released.
 
----
+```json
+{
+  "name": "my-project",
+  "version": "1.0.0",
+  "dependencies": {
+    "crds-styles": "crdschurch/crds-styles#development"
+  }
+}
+```
+
+In both of these cases, you'll need to import the stylesheet in your
+application's main '.scss' file using the following convention. You can
+optionally override any of the customizable Bootstrap variables prior to
+import.
 
-Once you've added `crds-styles` to `package.json` you need to import the stylesheet in your application's main '.scss' file using the following convention. You can optionally override any of the customizable Bootstrap variables prior to import...
+```scss
+@import '~crds-styles/assets/stylesheets/bootstrap';
+```
+
+_Note: The tilde character (`~`) assumes use of the
+[node-sass](https://github.com/sass/node-sass) library.
```

Comment:
Is this accurate? 

It's my understanding that the tilde character convention for resolving to `node_modules/` is provided via [Webpack's sass-loader](https://github.com/webpack-contrib/sass-loader#imports) plugin. I don't believe `node-sass` supports this feature out-of-the-box but you can use [node-sass-tilde-importer](https://github.com/matthewdavidson/node-sass-tilde-importer) package to emulate the same behavior.

Reply:
You're right. Good catch. Thanks.